templates/_macros/vf_rich-horizontal-list.jinja

```diff
@@ -0,0 +1,110 @@
+{% macro vf_rich_horizontal_list(
+  title_text,
+  variant="full-width",
+  list_item_style=""
+) -%}
+  {% set image_content = caller("image") %}
+  {% set description_content = caller("description") %}
+  {% set logo_section_content = caller("logo_section") %}
+  {% set cta_content = caller("cta") %}
+  {% set has_image = image_content|trim|length > 0 %}
+  {% set has_logo_section = logo_section_content|trim|length > 0 %}
+  {% set has_list = caller("list_item_1")|trim|length > 0 %}
+  {% set has_cta = cta_content|trim|length > 0 %}
+  {% set has_description = description_content|trim|length > 0 %}
+  {% set max_list_items = 8 %}
+
+  {% set list_item_style = list_item_style|trim|lower %}
+  {% set variant = variant|trim|lower|replace("/", "-") %}
+
+  {% if list_item_style|length > 0 and list_item_style not in ["bullet", "tick", "number"] %}
+    {% set list_item_style = "" %}
+  {% endif %}
+
+  {% if variant not in ["full-width", "50-50"] %}
+    {% set variant = "full-width" %}
+  {% endif %}
+
+  {% if variant == "50-50" %}
+    {% set right_column_contents_provided = has_description or has_logo_section %}
+  {% endif %}
+
+  {% if list_item_style == "bullet" %}
+    {% set list_item_class = "has-bullet" %}
+  {% elif list_item_style == "tick" %}
+    {% set list_item_class = "is-ticked" %}
+  {% endif %}
+
+  {% set list_element_type = "ul" %}
+  {% if list_item_style == "number" %}
+    {% set list_element_type = "ol" %}
+  {% endif %}
+
+  <div class="p-section">
+    <div class="grid-row">
+      {%- if has_image %}
+        {{- image_content -}}
+      {%- endif -%}
+      {#- 50/50 layout requires that content for the right column is passed in -#}
+      {%- if variant == "50-50" and right_column_contents_provided %}
+        <div class="grid-row">
+          {#- If there is a description, split the title and description. Otherwise, the title spans full width. -#}
+          <div class="grid-col-4{% if has_description %} grid-col-medium-2{% endif %}">
+            <h2>{{- title_text -}}</h2>
+          </div>
+          {%- if has_description %}
+            <div class="grid-col-4 grid-col-medium-2">
+              {{- description_content -}}
+            </div>
+          {%- endif -%}
+          {%- if has_logo_section %}
+            {#- When the description is present, the logo section wraps to another line, so we need to offset it to align with the description. -#}
+            <div class="grid-col-4{% if has_description %} grid-col-start-large-5{% endif %}">
+              {{- logo_section_content -}}
+            </div>
+          {%- endif -%}
+        </div>
+      {%- else %}
+        <h2>{{- title_text -}}</h2>
+        {%- if has_logo_section %}
+          {{- logo_section_content -}}
+        {%- endif -%}
+        {{- description_content -}}
+      {%- endif %}
```

Comment:
Description should precede the logo section on small in both variants; currently that is only the case for the 50/50 variant